Sources/ScryfallKit/Models/Card/Card+enums.swift

```diff
@@ -198,23 +198,25 @@ extension Card {
   }
 
   /// Effects applied to a Magic card frame
-  ///
+  /// 
   /// [Scryfall documentation](https://scryfall.com/docs/api/frames#frame-effects)
-  public enum FrameEffect: String, Codable, CaseIterable, Sendable {
+  public enum FrameEffect: RawRepresentable, Codable, Sendable, CaseIterable, Equatable, Hashable {
     case legendary, miracle, nyxtouched, draft, devoid, tombstone, colorshifted, inverted,
-      sunmoondfc, compasslanddfc, originpwdfc, mooneldrazidfc, waxingandwaningmoondfc, showcase,
-      extendedart, companion, etched, snow, lesson, convertdfc, fandfc, battle, gravestone, fullart,
-      vehicle, borderless, extended, spree, textless, unknown, enchantment, shatteredglass, upsidedowndfc
-
-    public init(from decoder: Decoder) throws {
-      self =
-        try FrameEffect(rawValue: decoder.singleValueContainer().decode(RawValue.self)) ?? .unknown
-      if self == .unknown, let rawValue = try? String(from: decoder) {
-        if #available(iOS 14.0, macOS 11.0, *) {
-          Logger.decoder.error("Decoded unknown FrameEffect: \(rawValue)")
-        } else {
-          print("Decoded unknown FrameEffect: \(rawValue)")
-        }
+         sunmoondfc, compasslanddfc, originpwdfc, mooneldrazidfc, waxingandwaningmoondfc, showcase,
+         extendedart, companion, etched, snow, lesson, convertdfc, fandfc, battle, gravestone, fullart,
+         vehicle, borderless, extended, spree, textless, enchantment, shatteredglass, upsidedowndfc
+    /// A layout that hasn't been added to ScryfallKit yet
+    case unknown(String)
+
+    /// All known Magic: the Gathering frame effects
+    public static let allCases: [Card.FrameEffect] = [
+      .legendary, .miracle, .nyxtouched, .draft, .devoid, .tombstone, .colorshifted, .inverted, .sunmoondfc, .compasslanddfc, .originpwdfc, .mooneldrazidfc, .waxingandwaningmoondfc, .showcase, .extendedart, .companion, .etched, .snow, .lesson, .convertdfc, .fandfc, .battle, .gravestone, .fullart, .vehicle, .borderless, .extended, .spree, .textless, .enchantment,
```

Comment:
Could we also take the opportunity to make these cases camelCase to align with all the other enums we have?